tasks/gulp/serve.js

```diff
@@ -0,0 +1,10 @@
+const gulp = require('gulp')
+const connect = require('gulp-connect')
```

Comment:
It's lighter weight, doesn't inject JavaScript into the running app, and doesn't need an additional port open.

Reply:
fair enough. just looked at the readme, it's moved on quite a lot since i last looked (including getting a sponsor!). looks like it now supports everything we'd want from browsersync anyway (namely livereload) 😄